networks.tf

```diff
@@ -1,43 +1,59 @@
+locals {
+  pcf_virtual_network_name = "${var.env_name}-virtual-network"
+  management_subnet_name   = "${var.env_name}-management-subnet"
+  pas_subnet_name          = "${var.env_name}-pas-subnet"
+  services_subnet_name     = "${var.env_name}-services-subnet"
+}
+
 resource "azurerm_virtual_network" "pcf_virtual_network" {
-  name                = "${var.env_name}-virtual-network"
+  name                = "${var.pcf_virtual_network_name != "" ? var.pcf_virtual_network_name : local.pcf_virtual_network_name}"
   depends_on          = ["azurerm_resource_group.pcf_resource_group"]
   resource_group_name = "${azurerm_resource_group.pcf_resource_group.name}"
   address_space       = "${var.pcf_virtual_network_address_space}"
   location            = "${var.location}"
 }
 
+<<<<<<< HEAD
+resource "azurerm_subnet" "management_subnet" {
+  name                      = "${var.management_subnet_name != "" ? var.management_subnet_name : local.management_subnet_name}"
+  depends_on                = ["azurerm_resource_group.pcf_resource_group"]
+  resource_group_name       = "${azurerm_resource_group.pcf_resource_group.name}"
+  virtual_network_name      = "${azurerm_virtual_network.pcf_virtual_network.name}"
+  address_prefix            = "${var.pcf_management_subnet}"
+=======
 resource "azurerm_subnet" "infrastructure_subnet" {
   name                 = "${var.env_name}-infrastructure-subnet"
   depends_on           = ["azurerm_resource_group.pcf_resource_group"]
   resource_group_name  = "${azurerm_resource_group.pcf_resource_group.name}"
   virtual_network_name = "${azurerm_virtual_network.pcf_virtual_network.name}"
   address_prefix       = "${var.pcf_infrastructure_subnet}"
+>>>>>>> 16d3bb5b449245d451e36d500fa431b27989c6e4
```

Comment:
There's a conflict here.

Reply:
This PR was for a specific use case and now that use case has vanished. Due to the recent overhaul of the repo and the vanishing use case, I'm not prioritizing the work for now. Go ahead and close it for now, if things change, we can revisit.

Reply:
Thanks for the update!